GitUI/CommandsDialogs/BrowseDialog/DashboardControl/DashboardCategory.cs

```diff
@@ -49,7 +49,7 @@ public DashboardCategory(string title, RepositoryCategory repositoryCategory)
 
         public RepositoryCategory RepositoryCategory
         {
-            get { return _repositoryCategory; }
+            get => _repositoryCategory;
```

Comment:
👎 I'm not a fan of mixing expression-bodied accessors and block-bodied accessors in the same property.

Reply:
I'm not so fussed. To me they're different members and it's less egregious than say an if/else block where one has braces and the other doesn't. Again, a matter of taste. I find the arrow and lack of `return` easier on the eyes.

Reply:
Agree with Sam - I prefer not to mix, looks inconsistent

Reply:
Fixed in 1887d7134.